arches/app/models/migrations/0005_4_0_1.py

```diff
@@ -17,12 +17,6 @@ def forwards_func(apps, schema_editor):
     # We get the model from the versioned app registry;
     # if we directly import it, it'll be the wrong version
 
-    # index base Arches concept
```

Comment:
@chiatt are these migration changes related to this PR or something else?

Reply:
They are actually related. Those calls to index concepts in migrations end up creating an old, invalid concept mapping which persists once the test database is created and migrated. Errors will occur if a term search is performed without first deleting the invalid concept mapping and creating a new one. I could have replaced the concept index in my test, but I wanted to avoid that because the reason for deleting and recreating the concept index would have been unclear to others.  So, it made sense to get rid of the invalid mapping in the migrations. That way, too, they won't haunt anyone else going forward.